frontend/src/components/AccountList.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="flex flex-col">
```

Comment:
I think it would be better, if this was the container directly. then the search bar and create account button could be moved into that header. this would ensure a better user experience, since these options are always available and not only when scolled all the way up

Reply:
you mean like this?

Reply:
is my github bugged or are there no changes here?

Reply:
if you presson this comment aou get the code, when you commented i think. you have to press on files at the top

Reply:
I see what you changed. I meant having the Account List like this:
```Vue
<template>
  <ContainerComponent>
    <template #Header>
      <div class=flex>
        <!-- SeachBar and Add Button -->
      </div>
    </template>
    <template #default>
      <!-- List -->
    </template>
  </ContainerComponent>
</template>
```

Reply:
should the add button really be in the accountlist? without we could may reuse it (i know we dont).  We could may just stick the addbutton at the bottom?

Reply:
From a sementic point of view it belongs into the container, like you already have it. and for the sake of user experience, the search bar and add button should both be in the header. So the other option would be to pull be search bar out and give the account list either an already filtered list or the string to filter by

Reply:
hm the searchbar and the createbutton are no longer visible now. you could may look why?

Reply:
In der accountList fehlt der import von der ContainerComponent

Reply:
should work now